### PR TITLE
Made Common its own class rather than a module

### LINKED ISSUE #19: Change subclassing of LinuxAdmin to composition by including LinuxAdmin::Common
If all we need is the behavior from LinuxAdmin::Common, we should just include the module into our classes instead of using inheritance.

### PR DESCRIPTION
**Pull request by carbonin**
Previously nearly all classes were mixing in LinuxAdmin::Common
to get access to what really are just utility methods.

Not sure when a good time to make this sort of change is, but I'm giving it a shot.
@bdunne @Fryguy @jrafanie 

Fixes #19

### CODE REVIEW

**Comment by jrafanie** [comment]
This makes sense to me.  :+1:

File: spec/physical_volume_spec.rb [hunk 1]
```diff
@@ -52,23 +52,23 @@
...
+        .with(LinuxAdmin::Common.cmd(:pvcreate),
+              :params => {nil => '/dev/hda'})
       described_class.create disk
     end
 
     it "returns new physical volume" do
-      allow(LinuxAdmin::VolumeGroup).to receive_messages(:run! => double(:output => ""))
-      allow(described_class).to receive_messages(:run! => double(:output => ""))
+      allow(LinuxAdmin::Common).to receive_messages(:run! => double(:output => ""))
+      allow(LinuxAdmin::Common).to receive_messages(:run! => double(:output => ""))
```
Comment (bdunne) [inline spec/physical_volume_spec.rb]: Do you need this twice?  `allow` should handle any number of calls, right?

File: spec/physical_volume_spec.rb [hunk 2]
```diff
@@ -52,23 +52,23 @@
...
       pv = described_class.create disk
       expect(pv).to be_an_instance_of(described_class)
       expect(pv.device_name).to eq('/dev/hda')
     end
 
     it "adds physical volume to local registry" do
-      allow(LinuxAdmin::VolumeGroup).to receive_messages(:run! => double(:output => ""))
-      allow(described_class).to receive_messages(:run! => double(:output => ""))
+      allow(LinuxAdmin::Common).to receive_messages(:run! => double(:output => ""))
+      allow(LinuxAdmin::Common).to receive_messages(:run! => double(:output => ""))
```
Comment (bdunne) [inline spec/physical_volume_spec.rb]: same

File: spec/physical_volume_spec.rb [hunk 3]
```diff
@@ -76,17 +76,17 @@
...
+        .and_return(double(:output => @physical_volumes))
+      expect(LinuxAdmin::Common).to receive(:run!).and_return(double(:output => @groups)) # stub out call to vgdisplay
       described_class.scan
     end
 
     it "returns local physical volumes" do
-      expect(described_class).to receive(:run!).and_return(double(:output => @physical_volumes))
-      expect(LinuxAdmin::VolumeGroup).to receive(:run!).and_return(double(:output => @groups))
+      expect(LinuxAdmin::Common).to receive(:run!).and_return(double(:output => @physical_volumes))
+      expect(LinuxAdmin::Common).to receive(:run!).and_return(double(:output => @groups))
```
Comment (bdunne) [inline spec/physical_volume_spec.rb]: You should be able to put both doubles in a single `and_return` and the first hit will get the first response and the second will get the second response.

File: spec/physical_volume_spec.rb [hunk 4]
```diff
@@ -95,8 +95,8 @@
     end
 
     it "resolves volume group references" do
-      expect(described_class).to receive(:run!).and_return(double(:output => @physical_volumes))
-      expect(LinuxAdmin::VolumeGroup).to receive(:run!).and_return(double(:output => @groups))
+      expect(LinuxAdmin::Common).to receive(:run!).and_return(double(:output => @physical_volumes))
+      expect(LinuxAdmin::Common).to receive(:run!).and_return(double(:output => @groups))
```
Comment (bdunne) [inline spec/physical_volume_spec.rb]: same

File: lib/linux_admin/common.rb [hunk 5]
```diff
@@ -1,25 +1,25 @@
 require 'awesome_spawn'
 
 module LinuxAdmin
-  module Common
+  class Common
```
Comment (bdunne) [inline lib/linux_admin/common.rb]: Do you need to change this?  I don't see us creating an instance anywhere and all the methods are class methods.

**Pull request merged**